frontend/src/app/projects/project-detail/model-overview/model-overview.component.ts

```diff
@@ -55,10 +55,6 @@ export class ModelOverviewComponent implements OnInit {
     if (primaryModel) {
       return primaryModel.path;
     } else {
-      this.toastService.showError(
-        "Couldn't open the Git repository",
-        'No primary Git model was found. Please contact your project lead or administrator.'
-      );
```

Comment:
Note for later: During the rebase, the commit removing this snippet must be reworked to prevent a conflict with the solution of the rebase source.